sherpa/astro/io/tests/test_io.py

```diff
@@ -0,0 +1,59 @@
+#
+#  Copyright (C) 2016  Smithsonian Astrophysical Observatory
+#
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License along
+#  with this program; if not, write to the Free Software Foundation, Inc.,
+#  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+
+from sherpa.utils import SherpaTestCase, has_fits_support, has_package_from_list
+from sherpa.astro import ui
+
+from unittest import skipIf
+from tempfile import NamedTemporaryFile
+import warnings
+
+
+class test_89_issues(SherpaTestCase):
+    def setUp(self):
+        ui.clean()
+
+    @skipIf(not has_package_from_list("sherpa.astro.xspec"), "xspec required")
```

Comment:
Do we gain anything from this test (i.e. `test_mod_fits`) given that the XSPEC tests have what looks to be a more-extensive test `text_xspec_tablemodel` https://github.com/sherpa/sherpa/blob/ca2db23857a4da09ca7fb3adb7b94b2871da67f3/sherpa/astro/xspec/tests/test_xspec.py#L305

I think the conditions for running both tests are the same (both fits and xspec) and the XSPEC version includes these tests - but I'm in a hurry and may have missed something.

If we do keep this test then don't we need to add a `@skipIf(test_data_missing)` decorator to make sure that the RCS.mod file is available?